### PR TITLE
Update requirements && Small refactor

### PR DESCRIPTION
**Pull request by SantaSpeen**

### CODE REVIEW

File: src/app/__init__.py [hunk 1]
```diff
@@ -42,11 +52,11 @@ def user_loader(uid):
 
         try:
             return User.query.get(int(uid))
-        except Exception:
+        except Exception as e:
             return None
 
 
-def create(name=None, cfg: object = ConfigProduction) -> Flask:
+def create(name: str = None, cfg: Type[ConfigProduction] or Type[ConfigDevelopment] = ConfigProduction) -> Flask:
```
Comment (kirillzhosul) [inline src/app/__init__.py:59]: Why not base class `Config`?

**Review by kirillzhosul — APPROVED** [review]
Seems fine but refactoring comment may be obscured or removed later. Nah, why not.

File: src/app/__init__.py (hunk 1 above)
Comment (SantaSpeen) [reply src/app/__init__.py:59]: Так надо

Comment (kirillzhosul) [reply src/app/__init__.py:59]: А по шапке?

**Pull request merged**